permamodel/components/bmi_frost_number.py

```diff
@@ -229,6 +233,29 @@ def update(self):
         self._model.calculate_frost_numbers()
         self._values['frostnumber__air'] = self._model.air_frost_number
 
+    def update_frac(self, time_fraction):
```

Comment:
One other thought...doesn't the BMI interface *require* that an update_frac() method exist?